content/applications/hr/fleet/configuration.rst

```diff
@@ -57,109 +57,127 @@ for existing available cars in this field.
+:guilabel:`(cloud with an upwards arrow)` icon.
 
 .. _fleet/configuration/models:
 
-Vehicle Models
-==============
+Models
+------
 
 Many manufacturers have a variety of models that they produce. When adding a vehicle to the fleet,
-it is important to specify the vehicle model being added. Odoo comes with pre-configured car models
-from four major auto manufacturers, and one pre-configured bicycle model from one bicycle
-manufacturer: Audi, BMW, Mercedes, Opel (cars), and Eddy Merckx (bicycle). If a vehicle *other* than
-the pre-configured models from these manufacturers are part of a fleet, the model (and/or
-manufacturer) will need to be added to the database.
-
-Preconfigured Models
---------------------
-
-The following models are pre-loaded in Odoo and do not need to be added to the database:
-
-+-------+--------------+-------------+-----------+---------------+
-| AUDI  | BMW          | Eddy Merckx | Mercedes  | Opel          |
-+=======+==============+=============+===========+===============+
-| A1    | Serie 1      | SanRemo76   | Class A   | Agilia        |
-+-------+--------------+-------------+-----------+---------------+
-| A3    | Serie 3      |             | Class B   | Ampera        |
-+-------+--------------+-------------+-----------+---------------+
-| A4    | Serie 5      |             | Class C   | Antara        |
-+-------+--------------+-------------+-----------+---------------+
-| A5    | Serie 6      |             | Class CL  | Astra         |
-+-------+--------------+-------------+-----------+---------------+
-| A6    | Serie 7      |             | Class CLS | AstraGTC      |
-+-------+--------------+-------------+-----------+---------------+
-| A7    | Serie Hybrid |             | Class E   | Combo Tour    |
-+-------+--------------+-------------+-----------+---------------+
-| A8    | Serie M      |             | Class GL  | Corsa         |
-+-------+--------------+-------------+-----------+---------------+
-| Q3    | Serie X      |             | Class GLK | Insignia      |
-+-------+--------------+-------------+-----------+---------------+
-| Q5    | Serie Z4     |             | Class M   | Meriva        |
-+-------+--------------+-------------+-----------+---------------+
-| Q7    |              |             | Class R   | Mokka         |
-+-------+--------------+-------------+-----------+---------------+
-| TT    |              |             | Class S   | Zafira        |
-+-------+--------------+-------------+-----------+---------------+
-|       |              |             | Class SLK | Zafira Tourer |
-+-------+--------------+-------------+-----------+---------------+
-|       |              |             | Class SLS |               |
-+-------+--------------+-------------+-----------+---------------+
+it is important to specify the vehicle model being added.
+
+Odoo comes with pre-configured car models from four major auto manufacturers: Audi, BMW, Mercedes,
```

Comment:
redundant?

Reply:
Yes- I took out the first part, and changed up the spacing.